compiler/rustc_mir/src/interpret/intrinsics.rs

```diff
@@ -322,6 +322,24 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
                 let result = Scalar::from_uint(truncated_bits, layout.size);
                 self.write_scalar(result, dest)?;
             }
+            sym::copy_nonoverlapping => {
```

Comment:
Don't you need to also implement `copy` and `move_val_init`?

See https://github.com/rust-lang/miri/blob/master/src/shims/intrinsics.rs for the Miri implementation.

Reply:
Oh, I forgot that I reverted `copy`. Thanks :)